package.json

```
@@ -17,7 +17,7 @@
   },
   "scripts": {
     "prepublishOnly": "shx chmod -R +x ./bin",
-    "preinstall": "yarn submodule && node build.js",
+    "preinstall": "node build.js",
```

Is there a reason we didn't change the `yarn <command>` alias to the equivalent `npm run <command>` alias?
```suggestion
    "preinstall": "npm run submodule && node build.js",
```

---

That gets run both in development and for an end-user install, so that will fails in a similar way for an end-user install of `npm i minidump`, results in:
```
fatal: not a git repository (or any of the parent directories): .git
```
I believe we need to not run `git submodule` at all during an end-user install (at best it silently does nothing, very likely it errors, at worst, it actually does a submodule command on an arbitrary parent module somewhere up the tree, I think?)